src/default.nix

```diff
@@ -82,7 +82,7 @@ let
     if url != null && !submodules then
       builtins.fetchTarball {
         inherit url;
-        sha256 = hash; # FIXME: check nix version & use SRI hashes
+        sha256 = hash;
```

Comment:
Can't we then also switch to `inherit hash` now? (same for all other `fetchTarball` invocations)

Reply:
Well, the interface of te builtins fetchers is all over the place, `fetchTarball` and `fetchurl` accept only `url`, `name` and `sha256` (which can be plain, or in SRI form), while `fetchGit` uses the `fetchTree` interface which is different and uses `narHash` for the hash